### ISSUE TITLE
Undocumented change to non-HTML escaped strings

### ISSUE THREAD

**Issue by RichardoC** [tidwall/gjson#365]
Can this behaviour please be documented and warned of? 

Current users might be relying on the sanitised return values and wouldn't be expecting this to change in a patch version.

<https://github.com/tidwall/gjson/commit/28d458b14c40df6321bba23bcb39579d468f5e97>

Thanks!

**Comment by tidwall** [comment]
You are right. After further consideration I change the behavior back to the previous and added the flag `DisableEscapeHTML` to provide users the option.
